src/remote_adapter/remote_adapter_client/qt_impl/sdl_remote_adapter_qt_client.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <QtCore>
+#include <QObject>
+
+#include <string>
+#include <utility>
+
+#include "sdl_remote_adapter_client.h"
+// #include "qt_impl/sdl_remote_adapter_receive_thread.h"
+
+namespace lua_lib {
+
+struct MqParams {
+    const std::string name;
+    const int max_messages_number;
+    const int max_message_size;
+    const int flags;
+    const int mode;
+};
+
+class SDLRemoteTestAdapterReceiveThread;
+
+class SDLRemoteTestAdapterQtClient : public QObject {
+Q_OBJECT
+
+public:
+    SDLRemoteTestAdapterQtClient(const std::string& host,
+                                 uint32_t port,
+                                 MqParams& in_params,
+                                 MqParams& out_params,
+                                 QObject* parent = Q_NULLPTR);
+
+    ~SDLRemoteTestAdapterQtClient();
+
+    // /**
+    // * @brief Connect client to server and open queue with default parameters
+    // * @param host server host to connect
+    // * @param port server port to connect
+    // */
+    // void connect(const std::string& host, uint32_t port, const std::string& name);
```

Comment:
@aderiabin if this is not used please delete commented code

Reply:
@EKuliiev @KVGrygoriev  Unused code was removed (see bc3f86577b740bc84201d7d0edaf82cce1d8ec52)